grafana_monitoring/cloud_datasource.yaml

```diff
@@ -0,0 +1,34 @@
+apiVersion: 1
+
+datasources:
+  - name: CloudInfluxDB
+    type: influxdb
+    access: proxy
+    uid: "openstack_grafana"
+    user: $PROD_INFLUX_USERNAME
+    url: "http://influx1.nubes.rl.ac.uk:8086"
+    #database: "cloud" # this is a deprecated field but works for v10
+    jsonData:
+      dbName : "cloud" 
+      httpMode: POST
+      timeInterval: "60"
+      maxSeries: 200
+    secureJsonData:
+      password: $PROD_INFLUX_PASSWORD
+
+  - name: grafana-power-metrics
+    type: grafana-opensearch-datasource
+    access: proxy
+    uid: "opensearch-grafana"
+    url: https://172.16.114.193:9200
```

Comment:
Could we have the URL be given by a variable, similar to how we use a variable for the user ?